cmd/servicelog/post.go

```diff
@@ -355,34 +355,26 @@ func printClusters(clusters []*v1.Cluster) (err error) {
 	return table.Flush()
 }
 
-func printTemplate() (err error) {
-	exampleMessage, err := json.Marshal(Message)
-	if err != nil {
-		return err
-	}
-	return dump.Pretty(os.Stdout, exampleMessage)
-}
-
-func createPostRequest(ocmClient *sdk.Connection, cluster *v1.Cluster) (request *sdk.Request, err error) {
+func createPostRequest(ocmClient *sdk.Connection, cluster *ocm_clustermgmt.Cluster) (request *ocm_servicelog.ClusterLogsAddRequest, err error) {
 	// Create and populate the request:
-	request = ocmClient.Post()
-	err = arguments.ApplyPathArg(request, targetAPIPath)
-	if err != nil {
-		return nil, fmt.Errorf("cannot parse API path '%s': %v", targetAPIPath, err)
-	}
+	request = ocmClient.ServiceLogs().V1().ClusterLogs().Add()
 
-	Message.ClusterUUID = cluster.ExternalID()
-	Message.ClusterID = cluster.ID()
-	if subscription := cluster.Subscription(); subscription != nil {
-		Message.SubscriptionID = cluster.Subscription().ID()
+	// make sure the builder is up to date
+	Message.Builder = Message.Builder.Copy(Message.LogEntry)
+
+	// add identifiers
+	Message.Builder = Message.Builder.ClusterUUID(cluster.ExternalID())
```

Comment:
is this really required? 
Can you not just do:
```suggestion
	Message.Builder.ClusterUUID(cluster.ExternalID())
```

Reply:
Don't think so, but maybe

Reply:
fixed and it works, so removed the unnecessary equal sign